src/main/webapp/lectures.js

```diff
@@ -14,6 +14,7 @@
 
 /* Used to gather URL parameters. */
 const URL_PARAM_ID = 'id';
+const URL_PARAM_VIDEO_ID = 'video-id';
```

Comment:
can these be renamed to PARAM_ to match names in other files?  We might want to create a const.js to have things in one place as part of #51